bin/node/runtime/src/lib.rs

```diff
@@ -401,7 +401,7 @@ impl pallet_indices::Config for Runtime {
 }
 
 parameter_types! {
-	pub const ExistentialDeposit: Balance = 1 * DOLLARS;
+	pub const ExistentialDeposit: Balance = 1 * DOLLARS / 100;
```

Comment:
Can you explain why we need to divide by 100 here?

Reply:
1 DOLLAR equals to 100 tokens. So I divided with 100 to make Existential Deposit 1 token.